LoopFollow/ViewControllers/AlarmViewController.swift

```diff
@@ -1660,7 +1660,7 @@ class AlarmViewController: FormViewController {
 
         <<< StepperRow("alertMissedReading") { row in
             row.title = "Time"
-            row.cell.stepper.stepValue = 5
+            row.cell.stepper.stepValue = 1
```

Comment:
I modified this to:
            row.cell.stepper.stepValue = 5
            row.cell.stepper.minimumValue = 11
            row.cell.stepper.maximumValue = 121

And did a test build. As soon as I tapped on the Time selector (+/-) it switch to allowing choices of 11, 16, 21, etc.

I think this is preferred over a picker of 1 minute.